mrbelvedereci/templates/layout_full.html

```diff
@@ -102,6 +102,22 @@ <h3 class="slds-text-heading--medium">Notifications</h3>
         </div>
         </a>
       </li>
+      {% if request.user.is_staff %}
```

Comment:
do we want to add the interactive API browser  as a button too?

Reply:
I'd prefer to keep it clean and not have a button for API.

Reply:
I agree with @cpolcyn for now.  I want to get the more granular restrictions set up on the API so we can open it up more.  Ideally there'd even be anonymous access to the API for read on public info that would be throttled via DRF.